dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractHostedEntryResource.java

```diff
@@ -361,6 +364,10 @@ public T deleteHostedVersion(@ApiParam(hidden = true) @Auth User user,
         checkEntry(entry);
         checkUserCanUpdate(user, entry);
         checkHosted(entry);
+        // If the version that's about to be deleted is the default version, unset it
+        if (entry.getActualDefaultVersion().getName().equals(version)) {
+            entry.setActualDefaultVersion(null);
```

Comment:
Thinking it might be better to just choose the newest version available and setting that to the default version (in the case that there is at least one left).

Reply:
Ok, that's do-able